newIDE/app/src/UI/KeyboardShortcuts/index.js

```diff
@@ -108,7 +108,10 @@ export default class KeyboardShortcuts {
     return this._spacePressed || this._mouseMidButtonPressed;
   }
 
-  shouldZoom() {
+  shouldZoom(evt: WheelEvent) {
+    // Browsers trigger a wheel event with ctrlKey or metaKey to true when the user
+    // does a pinch gesture on a trackpad. If this is the case, we zoom.
```

Comment:
Very interesting, I did not know! Is there a source about this somewhere?

Double check this is not blocking something like ability to scroll horizontally with a mouse + alt pressed?

Reply:
Added a link. I can't find an official documentation, but quite a few SO links or articles or Chromium issues talking about it.
It seems like it was a workaround all browsers introduced for devs to handle it correctly.

I've checked with a proper mouse and with other buttons pressed, and the behavior is what you expect I think 
(wheel does vertical scroll, shift or alt + wheel does horizontal scroll, cmd or ctrl + wheel does zoom)